R/git.R

```diff
@@ -0,0 +1,22 @@
+#' Git information
+#' 
+#' Displays git information from the repository
+#' 
+#' @import git2r
+#' @import shiny
+#' @export
+git_info <- function() {
+  print(getwd())
+  repo <- init(path = ".", bare = FALSE)
+  branch <- repository_head(repo)[[1]]
+  if (is.null(branch)) {
+    git_message <- "Empty repo."
+  } else {
+    last_commit <- as.character(commits(repo)[[1]])[[4]]
```

Comment:
Are you sure `[[1]]` and `[[4]]` will always work?

Reply:
`[[1]]` I'd leave as this is just a first commit from the top of the history list. `[[4]]` I changed to field name of the list.